surfactant/cmd/createconfig.py

```diff
@@ -0,0 +1,37 @@
+import json
+from pathlib import Path
+
+import click
+
+
+@click.command()
+@click.argument("directory", type=click.Path(exists=True))
+@click.option(
+    "-o",
+    "--output",
+    type=str,
+    help="Output JSON file name, defaults to end of directory string passed as input",
+)
+def create_config(directory, output):
+    """Create surfactant input configuration file based on input directory."""
+    starting_directory = Path(directory)
+    extract_paths = []
+
+    # Check if there are any files in the directory
+    if any(item.is_file() for item in starting_directory.iterdir()):
+        extract_paths.append(str(starting_directory).replace("\\", "/"))
```

Comment:
```suggestion
        extract_paths.append(starting_directory.as_posix())
```

`as_posix()` combines path components using `/` and returns the resulting path as a `str`.